### DISCUSSION TITLE
5.5?

### DISCUSSION THREAD

**Discussion by muchogrande** [NonStaticGH/CPathHostProject#15] [Q&A]
Hi, just wondering if you plan an official Fab release for 5.5? Still very much looking forward to the pro version - I absolutely love this plugin, for an indie dev it truly feels like a superpower!

**Comment by muchogrande** [comment]
FYI, Recompile works. Really hope the pro version is still in the works - this system is a godsend for this particular indie dev!

**Reply by SamStormSe** [reply]
hello, i have been trying to recompile it for 5.5 for many days but still coudlnt, can u tell me how to do it? or sending me the plugin file?

**Reply by muchogrande** [reply]
Are you recompiling via a C project? That's the only way it has ever worked for me (the popup recompile you get when trying via a BP only project rarely ever works). Recommend understanding how to do that if you don't know. 

But here is my recompile - should work (hope NonStaticGH doesn't mind) : https://www.dropbox.com/scl/fi/2pyop8fv355ir6uulzkw2/CPathfinding.rar?rlkey=26gurgoguxysl0zfqmjpjo4lg&dl=0

**Comment by SamStormSe** [comment]
yes i compiled with c project and even in console batch file but none of
them worked, i tried all the ways that i found on youtube and....,
whatever, thank you so much bro, this plugin is really life saving for us
as indie developers, thank you so much.

**Reply by muchogrande** [reply]
Strange, I think all I did was re-compile it. Oh wait, I may have changed a 5.4 to 5.5 in one of the files first. But yeah, agreed - there's nothing like this out there - I'm sad to see the dev appears to have stepped away from it.

**Comment by SamStormSe** [comment]
i tried everything, and when i try to build solution i will get some errors
like this. i couldnt find any solution for this, and i am not good at c++ ,
i am just developing with blueprints.

**Comment by SamStormSe** [comment]
maybe its bcz of my (vs idle) or something like this,nevermind, thank you
bro, have a good day or good night,if u have any problem with game
mechanics with blueprints especially for 3rd person medieval or 2.5d or
anything in level design, i can help u , i have more than 5 y experience on
it,

**Reply by muchogrande** [reply]
No sweat and thanks man, appreciate the offer. I'm solo-devving a 2.5D atm - all good besides a geometry script issue that's driving me up the wall. You on discord?

**Comment by SamStormSe** [comment]
yes i have discord, send me the link
